Ray.jl/src/function_manager.jl

```diff
@@ -27,6 +27,30 @@
 using ray_core_worker_julia_jll: JuliaGcsClient, Exists, Put, Get,
                                  JuliaFunctionDescriptor, function_descriptor
 
+const FUNCTION_SIZE_WARN_THRESHOLD = 10_000_000
+const FUNCTION_SIZE_ERROR_THRESHOLD = 100_000_000
+
+_mib_string(len) = string(div(len, 1024 * 1024), " MiB")
+
+function check_oversized_function(serialized, function_descriptor)
+    len = length(serialized)
+    check_msg = "Check that its definition is not implicitly capturing a large " *
+                "array or other object in scope. Tip: use `Ray.put()` to put large " *
+                "objects in the Ray object store."
```

Comment:
Are these messages based upon a message from Python Ray? May be good to include a reference here

Reply:
Should be made into a global constant